Roslyn.sln

```diff
@@ -410,6 +410,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.Exte
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Net.Compilers.Toolset.Package", "src\NuGet\Microsoft.Net.Compilers.Toolset\Microsoft.Net.Compilers.Toolset.Package.csproj", "{A74C7D2E-92FA-490A-B80A-28BEF56B56FC}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.ExternalAccess.Xamarin", "src\Tools\ExternalAccess\Xamarin\Microsoft.CodeAnalysis.ExternalAccess.Xamarin.csproj", "{6707896F-6F69-4DA0-9DD2-F1C6FB4BB939}"
```

Comment:
💡 Since this will be loaded in OOP scenarios, we should probably be clear with the name. Also we may want a Xamarin assembly for non-OOP scenarios later.

```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote", "src\Tools\ExternalAccess\Xamarin.Remote\Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote.csproj", "{6707896F-6F69-4DA0-9DD2-F1C6FB4BB939}"
```

Reply:
Addressed